lib/LTP/utils.pm

```diff
@@ -309,6 +309,9 @@ EOF
     script_run 'grep -e Huge -e PageTables /proc/meminfo';
     script_run 'echo 1 > /proc/sys/vm/nr_hugepages';
     script_run 'grep -e Huge -e PageTables /proc/meminfo';
+    script_run('echo NO_RUN_TO_PARITY > /sys/kernel/debug/sched/features')
+      if is_s390x() && is_sle_micro('6.2+') &&
```

Comment:
sle16 will face same behavior, kernel is shared  with micro

Reply:
We can add `|| is_sle('16+')` once the first failure shows up. Until then, I'd rather leave the condition as is.